test/drop_test/action.py

```diff
@@ -698,6 +698,7 @@ def __init__(
 
     async def run(self, drop: ffi.Drop):
         transfers = json.loads(drop.get_transfers_since(self._since_timestamp))
+        print(f"Transfers JSON:\n{json.dumps(transfers, indent=2)}")
```

Comment:
Is this useful to leave?

Reply:
Yeah, I mean it prints the DB contents. I find it more useful than not

Reply:
Okey dokey